### ISSUE TITLE
Inquiries about ursa progress

### ISSUE THREAD

**Issue by le520** [LineageOS/android_device_xiaomi_ursa#1]
Hello developer, I am a user of ursa equipment, and today I was pleasantly surprised to find that this model exists in the LineageOS warehouse. I want to ask, is there a LineageOS ROM available for this model?

**Comment by kartun83** [comment]
Hello, any news ?

**Comment by le520** [comment]
> Hello, any news ?

Yes, now you can build usable LineageOS and Aospa. But FOD still does not work.

**Comment by kartun83** [comment]
>> But FOD still does not work.
This means fingerprint scanner?
Any guides how to build it or may be link to prebuilt image ?

**Comment by le520** [comment]
> > > But FOD still does not work.
> > > This means fingerprint scanner?
> > > Any guides how to build it or may be link to prebuilt image ?

You can install telegram and search Nooralls by username, I will invite you to test this.
